ocpmodels/trainers/simple_trainer.py

```diff
@@ -65,7 +65,7 @@ def __init__(
         )
 
         self.load()
-        print(yaml.dump(self.config, default_flow_style=False))
+        # print(yaml.dump(self.config, default_flow_style=False))
```

Comment:
Should we uncomment this?

Reply:
I don't know, do you like it printed? I am happy to change it back, but it was printing stuff that is passed directly so it seemed unnecessary. Is there a point besides for debugging? Also, if in Jupyter can access self.config directly.

Reply:
I uncommented it for now